src/main/java/org/sonatype/nexus/ci/config/Nxrm3Configuration.groovy

```diff
@@ -69,5 +70,36 @@ class Nxrm3Configuration
         error(e, 'Nexus Repository Manager 3.x connection failed')
       }
     }
+
+    @Override
+    FormValidation doCheckServerUrl(@QueryParameter String value) {
+      def validation = super.doCheckServerUrl(value)
+
+      if (validation.kind != OK) {
+        return validation
+      }
+
+      // check nexus version, warn if < 3.13.0 PRO
+      try {
+        def response = new XmlSlurper().parseText(new URL("${value}/service/rest/wonderland/status").text)
+        def edition = response.edition.text()
+        def version = response.version.text()
+        def (major, minor) = version.tokenize('.').take(2).collect { it as int }
+
+        if (!edition.equalsIgnoreCase('pro') || major < 3 || minor < 13) {
+          return FormValidation.warning(
+              "NXRM ${edition} ${version} found. Some operations require a Professional Edition Nexus Repository " +
+                  "Manager server of version 3.13.0 or newer; use of an incompatible server will result in failed " +
```

Comment:
Version number shown in the String could be constructed rather than copied into this String, otherwise it complicates future update messaging

Reply:
I'm not sure I understand. Can you provide an example?

Reply:
If the version info is variables you can format the string, rather than having the numbers in two places.
```
static final int major = 3
static final int minor = 13
static final int patch = 0
String message = "whatever: " + [major, minor, patch].join('.')
```